src/wp-content/themes/twentynineteen/inc/back-compat.php

```diff
@@ -33,7 +33,7 @@ function twentynineteen_switch_theme() {
  *
  * @since Twenty Nineteen 1.0.0
  *
- * @global string $wp_version WordPress version.
+ * @global string $wp_version The WordPress version string.
```

Comment:
I had originally added new tags to match "WordPress version" in the `back-compat` files for consistency within the themes collection. Because other Core files repeatedly use "The WordPress version string," the themes match that instead now.